lib/HTFeed/GlacierZipAudit.pm

```diff
@@ -32,39 +33,39 @@ sub choose {
   my $path_prefix = shift || 's3://ht-deep-archive-backup/';
 
   my $dbh = HTFeed::DBTools->get_dbh();
-  my ($namespace, $objid, $path, $version);
-  my $sql = 'SELECT namespace,id,path,version FROM feed_backups' .
+  my ($namespace, $objid, $path, $version, $storage_name);
+  my $sql = 'SELECT namespace,id,path,version,storage_name FROM feed_backups' .
             " WHERE path LIKE '$path_prefix%'" .
             ' AND deleted IS NULL' .
             ' AND saved_md5sum IS NOT NULL' .
             ' AND restore_request IS NULL' .
             ' ORDER BY RAND() LIMIT 1';
   eval {
-    ($namespace, $objid, $path, $version) = $dbh->selectrow_array($sql);
+    ($namespace, $objid, $path, $version, $storage_name) = $dbh->selectrow_array($sql);
   };
   if ($@) {
     die "Database query failed: $@";
   }
   return { namespace => $namespace, objid => $objid,
-           path => $path, version => $version };
+           path => $path, version => $version,
+           storage_name => $storage_name };
 }
 
 # Get the objects for which we have issued restore requests
 sub pending_objects {
   my $path_prefix = shift || 's3://ht-deep-archive-backup/';
 
   my $dbh = HTFeed::DBTools->get_dbh();
-  my ($namespace, $objid, $path, $version);
-  my $sql = 'SELECT namespace,id,path,version FROM feed_backups' .
+  my $sql = 'SELECT namespace,id,path,version,storage_name FROM feed_backups' .
             " WHERE path LIKE '$path_prefix%'" .
```

Comment:
I think this is the main thing we'll want to change to condition on `storage_name` instead of `$path_prefix`

Reply:
`storage_name` set to `s3-deep-archive-mlibrary` by default, or would you prefer any storage name kept out of the codebase entirely and only passed in from `bin/some_script.pl`?

Reply:
Let's keep it out of the codebase entirely and pass it as an argument to `bin/some_script.pl`